.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Linux test
+
+on:
+    push:
+        branches: [ "*" ]
+    pull_request:
+        branches: [ "master" ]
```

Comment:
I noticed this only applies to master, do we actually want this?

Reply:
Does fast64 have other branch that master that will not make a pr to master ?

Reply:
Not currently no, but that's another reason to just apply to all branches (because if its a pr to fast64, it doesn't matter because there is only main, and if its a pr to a branch of a fork it will still run)

Reply:
Ok then will change that once I have the possibility